ui/app/components/clients/history.js

```diff
@@ -2,49 +2,29 @@ import Component from '@glimmer/component';
 import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
 import { tracked } from '@glimmer/tracking';
-import { isSameMonth, isAfter, isBefore } from 'date-fns';
+import { isAfter, isBefore, isSameMonth, format } from 'date-fns';
 import getStorage from 'vault/lib/token-storage';
-import { ARRAY_OF_MONTHS } from 'core/utils/date-formatters';
-import { dateFormat } from 'core/helpers/date-format';
 import { parseAPITimestamp } from 'core/utils/date-formatters';
 
-const INPUTTED_START_DATE = 'vault:ui-inputted-start-date';
-
+// TODO CMB: change class and file name to Dashboard
```

Comment:
leaving this TODO until after PR review so that the git diff is easier to follow